stdlib/Markdown/src/render/terminal/render.jl

```diff
@@ -70,15 +70,40 @@ function term(io::IO, f::Footnote, columns)
     end
 end
 
-function term(io::IO, md::List, columns)
+const _list_bullets = ["• ", "➤ ", "– "]
```

Comment:
Maybe imitate what GitHub uses?
```suggestion
const _list_bullets = ["• ", "◦ ", "▪︎ "]
```